fdbserver/masterserver.actor.cpp

```diff
@@ -241,7 +241,11 @@ void updateLiveCommittedVersion(Reference<MasterData> self, ReportRawCommittedVe
 	if (req.version > self->liveCommittedVersion.get()) {
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR && req.writtenTags.present()) {
 			// TraceEvent("Received ReportRawCommittedVersionRequest").detail("Version",req.version);
-			self->ssVersionVector.setVersion(req.writtenTags.get(), req.version);
+			int8_t primaryLocality = tagLocalityInvalid;
```

Comment:
this can be a field in `MasterData` so that we can blindly call `setVersion` here.

Reply:
Yes, makes sense.

Reply:
Done.